packages/vats/src/nameHub.js

```diff
@@ -335,7 +335,7 @@ export const prepareNameHubKit = zone => {
             if (pmap.has(key)) {
               // Reject only if already exists.
               const old = NonNullish(pmap.get(key));
-              old.reject(Error(`Value has been deleted`));
+              old.reject(Error(`Value for ${key} has been deleted`));
```

Comment:
:1st_place_medal:

Reply:
Is the `agoricNames` vat being upgraded in u19?

Is `nameHub` used in any other vats? Oh yeah... provisioning holds namesByAddress. Are we upgrading that one?

Reply:
I put notes in the Upgrade Considerations above.